backend/account/authentication_controller.py

```diff
@@ -173,6 +178,15 @@ def set_user_organization(self, request: Request, organization_id: str) -> Respo
                             {ErrorMessage.DUPLICATE_API}"
                     )
             self.create_tenant_user(organization=organization, user=user)
+
+            if new_organization:
+                try:
+                    self.auth_service.frictionless_onboarding(
+                        organization=organization, user=user
+                    )
+                except MethodNotImplemented:
+                    Logger.info("Method not implemented")
```

Comment:
NIT: Can be more explicit and mention which method is not implemented

Reply:
addressed